third_party/move/move-compiler/src/parser/lexer.rs

```diff
@@ -478,6 +478,25 @@ fn find_token(
                 (get_name_token(&text[..len]), len)
             }
         },
+        '"' => {
+            let line = &text.lines().next().unwrap()[1..];
+            match get_string_len(line) {
+                Some(_last_quote) => {
+                    let loc = make_loc(file_hash, start_offset, start_offset);
+                    return Err(Box::new(diag!(
+                        Syntax::InvalidByteString,
+                        (loc, "String literal must begin with b\" (for a byte string) or h\" (for a hex string)")
```

Comment:
Hex strings are `x"255"` according to the Move book, not `h"255"`. Fix here if this was a typo, else, we need to fix the Move book to say we also allow `h"255"`.

Reply:
Done.